lib/deps.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const kModuleError = Symbol('Error Loading Module');
+
+function makeErrorModule(error) {
+  return new Proxy(
+    { [kModuleError]: error },
```

Comment:
not a huge deal, but this means that `[kModuleError]` will _always_ be present on the proxy, though it may have a nullish value. This doesn't break any of your checks in the PR here, but the safest way would be to do this conditionally before creating the proxy:
```
function makeErrorModule(error) {
   const props = error ? { [kModuleError]: error } : {};
   return new Proxy(props, ...);
}
```

Reply:
In fact I don't need to define it as a prop at all! everything goes through the get: function so I'll just drop it as a prop